src/main/webapp/js/components/user-managment/roles.jsx

```diff
@@ -1,14 +1,75 @@
-import React from 'react';
-
-const Roles = () => (
-  <div className="container-fluid">
-    <h1>
-      Roles Roles Roles Roles
-      <br />
-      Placeholder Placeholder Placeholder Placeholder
-      <br />
-    </h1>
-  </div>
-);
-
-export default Roles;
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import EntityTable from '../entities/entity-table';
+import { ROLE_ENTITY, PERMISSION_ENTITY } from '../../utils/entity-types';
+
+export const defaultRoleConfig = [{
```

Comment:
rename this to roleFieldConfig and create a new file named 'field-configs.js' (in utils) and move this config there